### PR TITLE
docs - readme/general docs edits

### PR DESCRIPTION
**Pull request by thisisshi**
minor edits to capitalization, adding AWS to AWS resources in prep for future multi-cloud readme doc updates

### CODE REVIEW

**Comment by kapilt** [comment]
so i think it would be worthwhile doing this as group edit during the sprint, as we'll have several folks there working on multi-cloud support.

File: docs/source/usecases/s3globalgrants.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ S3 - Global Grants
 ==================
 
 Scan buckets that allow for global access in their
-ACLs and report them and the ACL permissions granted.
+ACLs and delete the associated ACL permissions.
```
Comment (thisisshi) [inline docs/source/usecases/s3globalgrants.rst]: per gitter convo, this doc wasn't clear on if the global-grant was being deleted

**Comment by thisisshi** [comment]
agreed, but should that happen now? azure/gcp support is still in the early stages. At this point, these changes mainly address some outstanding issues various users have had with the current docs, perhaps another PR to decouple AWS specific documentation would be better once azure or gcp support is available in a more GA state.

**Comment by kapilt** [comment]
agreed, let's merge as is, we can tackle the provider breakdown at a later date.

**Pull request merged**